README.md

```diff
@@ -0,0 +1,30 @@
+# Loopchain Tools
+ **Loopchain tools** is a Loopchain tool.
+
+The tool is consisted of two parts:
+- [Basictool]: provides configure preset files and supports variable API methods for Loopchain
+- [Gtool]: provides CLI monitoring services.
+
+## Getting Started
+### Installation
+ Loopchain Tools has no dependencies on Loopchain itself.
```

Comment:
How can I run Loopchain Tools itself, if it does not require Loopchain?

Can you explain?

Reply:
This branch is just copy of private One. So It needs another work to making it public. If you want it in this PR?

Reply:
I just made copy one and add a  new single feature. I didn't touch anything.

Reply:
You want it? retouch everything?

Reply:
That was not  my Goal.

Reply:
But I will reconsider it.